dom/telephony/USSDSession.cpp

```diff
@@ -98,7 +98,7 @@ USSDSession::Send(const nsAString& aUssd, ErrorResult& aRv)
     return nullptr;
   }
 
-  nsCOMPtr<nsITelephonyCallback> callback = new TelephonyCallback(promise);
+  nsCOMPtr<nsITelephonyDialCallback> callback = new TelephonyDialCallback(mWindow, nullptr, promise);
```

Comment:
We need TelephonyDialCallbacks instead of TelephonyCallbacks because USSDSession.send() needs to behave the same way as mozTelephony.dial(), in terms of promise solving.
With TelephonyCallback, NotifyDialMMISuccessXXX() and NotifyDialMMIError() were not available